docs/guide/repository.md

```diff
@@ -31,7 +31,7 @@ The Public API exposes Symfony services for all of its Repository services.
 Every Public API Service interface and value object defined in `eZ\Publish\API` namespace strictly follows [Semantic Versioning](https://semver.org/) backward compatibility (BC) promise for API consumers.
 It means that every usage of API (API call) is guaranteed to work between minor releases.
 
-What can change between minor releases is API method signature. Because of that, implementation of API interfaces by third party packages (except the ones implemented by eZ Systems bundles) is not directly supported.
+What can change between minor releases is the API method signature. Because of that, implementation of API interfaces by third party packages (except for the ones implemented with bundles) is not directly supported.
```

Comment:
```suggestion
What can change between minor releases is the API method signature. Because of that, implementation of API interfaces by third party packages (except for the ones implemented with built-in bundles) is not directly supported.
```